### DISCUSSION TITLE
绳网社区详细规则与准则

### DISCUSSION THREAD

**Discussion by nostalgiatan** [share121/inter-knot#4628] [其他]
﻿### 绳网社区详细规则与准则

![IMG_20240818_174044.jpg](https://github.com/user-attachments/assets/2eee9bc0-f513-4cce-bb1e-e34da6edac36)

 #### 1. 论坛定位与基本准则

 - **不碰政治：** 本社区严禁讨论敏感的政治议题，避免引发不必要的争议。
 - **不碰黄赌毒：** 社区内不得出现与色情、赌博、毒品相关的任何内容。
 - **不乱带节奏：** 杜绝任何形式的煽动性言论，保持论坛的和谐氛围。

 ### 1.1 带节奏行为的判定

 1. **帖子属性：**
    - **和平帖：** 内容客观、无攻击性的帖子。
    - **攻击帖：** 含有挑衅、讽刺或负面情绪的帖子。
   
 2. **和平帖中的带节奏行为：** 若和平帖下的评论出现带节奏行为，将由带节奏的一方承担责任，并予以相应处罚。
   
 3. **攻击帖中的带节奏行为：** 如果攻击性质的帖子引发进一步的负面互动，发帖者及参与负面互动的用户将共同承担责任。

 4. **严重事件：** 如冲突升级至多个用户间的对抗，所有参与方均需承担相应责任。

 5. **煽动性言论及不尊重行为：**
    - **禁止发布煽动性言论：** 包括但不限于鼓吹暴力、仇恨言论、歧视性言论等可能引起社会不稳定因素的言论。
    - **禁止侮辱、诽谤及恶意攻击：** 不得使用粗俗、侮辱性的语言或表达方式来针对其他用户或公众人物。
    - **禁止人身攻击：** 避免基于个人特征（如种族、性别、宗教信仰、性取向等）的攻击性言论。
    - **禁止挑衅与骚扰：** 不得发起无端的挑衅或持续骚扰其他用户。
    - **禁止恶意举报：** 不得滥用举报机制，故意对其他用户的正常发言进行恶意举报。
    - **禁止性暗示：** 本社区禁止任何形式的性暗示内容，包括但不限于隐晦的性挑逗、具有性暗示的词语或图片、以及可能引起性联想的描述等。

 6. **煽动性言论的处理：** 发布煽动性言论（包括性暗示）的用户将受到警告、删除违规内容等处罚。

 #### 2. 行为规范

 - **禁止语言攻击：** 不得以任何形式对其他用户进行言语上的攻击。
 - **禁止带节奏：** 不得在帖子中或评论区故意引导负面情绪，引发不必要的争端。
 - **禁止人身攻击：** 对其他用户进行诽谤、造谣或恶意揣测的行为均被视为违规。

 #### 3. 内容管理

 ### 3.1 色情内容的判定

 - **正常判定：**
   - 明确的性暗示或动作。
   - 特定身体部位的强调与暴露。
   - 极度暴露或半裸的视觉呈现。
 - **色情判定：**
   - 明确描绘性行为的内容。
   - 直接展示性器官的接触与互动。
   - 任何可能引起不适的性相关内容。
 - **性暗示定义：** 本社区禁止任何形式的性暗示内容，包括但不限于隐晦的性挑逗、具有性暗示的词语或图片、以及可能引起性联想的描述等。

 ### 3.2 交易与广告

 - **定义：**
   - **商业推广信息：** 包括任何形式的商业推广性质的活动链接（如促销、优惠券、邀请码等）、二维码引导下载或买卖、个人联系方式（如QQ号、微信号等）用于私下交易，以及欺骗性诱导外链。
   - **交易规定：** 在社区内进行的交易活动必须遵循既定的流程，禁止私下交易；禁止涉及送号、卖号、抽奖送粉丝、送福袋、捐赠、录屏抽奖等活动，除非事先获得官方批准且活动完全公开透明。

 - **后果：** 违反上述规定的用户将面临警告、删帖等处罚。

 ### 3.3 干扰社区秩序行为管理

 - **定义：**
   - **刷屏行为：** 连续发布重复内容或大量无关紧要信息的行为。此类行为将受到警告、封禁等处罚。
   - **擦边加虚拟信息：** 故意发布误导性信息或利用虚假身份进行欺骗的行为。

 - **后果：** 对于以上行为，违规者将面临警告、删帖等处罚措施。

 ### 3.4 法律遵守与公共讨论管理

 - **法律依据：** 禁止发布违反国家法律法规的内容。
 - **禁止内容：** 
   - 违反国家安全、民族仇恨、淫秽、诽谤等违法信息。
   - 未经资质，不得发布时政、军事、民生等敏感内容。
   - 禁止攻击性、诋毁性、煽动性言论针对国家政府。
   - 禁止可能导致公众对政府产生不信任的内容。

 ### 3.5 维护社会稳定与公共秩序

 - **禁止散布谣言与误导性信息：** 违规者将受处罚。
 - **不得扰乱社会秩序：** 禁止煽动暴力、散布仇恨等行为。
 - **不得破坏社会稳定：** 禁止发布不利社会稳定的内容。
 - **违规处理：** 违规者将受警告、删帖等处罚。

 #### 4. 安全、健康与尊重

 - **禁止恐怖主义、危险物品、毒品、赌博内容。**
 - **禁止自残、伤害他人、侮辱英烈、否定英烈事迹。**
 - **禁止美化侵略战争行为。**

 #### 5. 用户反馈

 - **途径：** 用户可以通过评论区或更新的日志进行反馈。
 - **作用：** 所有反馈都将被认真考虑，并用于规则的持续改进。

**Comment by LMA936** [comment]
支持<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >

**Reply by SAJN8455** [reply]
+1

**Comment by lantainAirblue** [comment]
文明绳网从我做起

**Comment by ID104772508** [comment]
支持

**Comment by Bisaqulaor** [comment]
支持，不然只会污染社区

**Comment by x18144506** [comment]
文明绳网从我做起

**Comment by Robin-txdy** [comment]
支持

**Comment by JiuXiat** [comment]
支持，保护社区环境人人有责

**Comment by wlmask** [comment]
支持!支持! 文明绳网从我做起

**Comment by szxtj** [comment]
加油

**Comment by arturiavr** [comment]
加油

**Comment by Minapocalis** [comment]
Civilization rope network，Excuse me, is this place taken over by the Sheriff's Bureau? I never thought it was really over，My rope net!!
!
![img_-375810294_1707339301538](https://github.com/user-attachments/assets/792eff50-f10c-47da-a59b-e79b4a77254c)

**Comment by FeiXiao-zm** [comment]
文明绳网从你我做起

**Comment by FAKERqx** [comment]
支持

**Comment by xiaoliaoggg** [comment]
支持！！
![Uploading mmexport1726318771869.jpg…]()

**Comment by lnh602** [comment]
发帖应该强制发图，不然瀑布屏一片漆黑，不好看

**Comment by kak-mono** [comment]
支持

**Comment by TianyiLuo0721** [comment]
支持！

**Comment by oldzhunotold** [comment]
支持

**Comment by si-xue610** [comment]
绳网它好像没有审核，所以才有这些东西的，绳网的作者是b站上的一个up主，目前我不知道ta有没有加，有的话就查严一点，如果没有.......快加啊！！！

**Comment by seoduer512** [comment]
文明绳网，从我做起

**Comment by Gentle-nature** [comment]
那那个想上匿叶龙的怎么说🙄

**Comment by kak-mono** [comment]
支持
